libraries/core_libs/network/include/network/tarcap/packets_handlers/get_votes_sync_packet_handler.hpp

```diff
@@ -21,6 +21,8 @@ class GetVotesSyncPacketHandler final : public ExtVotesPacketHandler {
 
   std::shared_ptr<PbftManager> pbft_mgr_;
   std::shared_ptr<NextVotesManager> next_votes_mgr_;
+
+  const uint32_t kMaxVotesInPacket{1000};
```

Comment:
shouldnt we check this limit when receiving votes packet ?

Reply:
Yes, I will add that.